ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -128,6 +129,7 @@ export default class ConfirmPageContainerContent extends Component {
       nativeCurrency,
       networkName,
       showBuyModal,
+      showingHardwareConnectionContents,
```

Comment:
Might be good to generalize this variable name. Within this component, this variable just determines whether we should show `ConfirmPageContainerSummary` or not. So we could change this to `hideConfirmPageContainerSummary`

Reply:
I prefer not to name things with a negative word; also, `showingHardwareConnectionContents` is akin to `showBuyModal`, representing whether or not we are showing the popover.  Maybe you'd prefer `showingHardwareConnectionPopover`?

Reply:
I was mostly thinking to remove the reference to the hardware wallet. In the context of this file, the hardware wallet is irrelevant, the boolean property just indicates whether or not to show `ConfirmPageContainerSummary`... the parent of this component can decide to hide or show that for whatever reason it chooses (not just because of the hardware wallet being connected)